### PR TITLE
remove codecgen from .tool-versions

### PR DESCRIPTION
**Pull request by krehermann**

### CODE REVIEW

**Review by eutopian — APPROVED** [review]

**Review by jmank88 — DISMISSED** [review]

**Review by krehermann — COMMENTED** [review]
had to push again to get CI out of a stuck loop failing on an unrelated test

**Review by jmank88 — APPROVED** [review]

**Review by eutopian — APPROVED** [review]

File: .tool-versions [hunk 1]
```diff
@@ -6,4 +6,3 @@ helm 3.10.3
 zig 0.10.1
 golangci-lint 1.51.2
 shellspec 0.28.1
-codecgen 1.2.10
```
Comment (prashantkumar1982) [inline .tool-versions:9]: Just curious, why was this needed?

Comment (krehermann) [reply .tool-versions:9]: i added it mistakenly because it seemed like all our tools were listed here. i'm not sure what this file is intended to do

**Review by prashantkumar1982 — APPROVED** [review]
Assuming that since @jkongie approved and merged the underlying wsrpc changes, this should be good to go.

**Review by eutopian — APPROVED** [review]

File: .tool-versions (hunk 1 above)
Comment (eutopian) [reply .tool-versions:9]: ahh... maybe we should add this to the readme somewhere; `.tool-versions` is the file for asdf managed dev tools https://asdf-vm.com/ to help ensure version consistency, but it also only works for supported packages

**Review by jmank88 — APPROVED** [review]
